__tests__/app.test.js

```diff
@@ -147,16 +189,14 @@ describe("3: PATCH /api/articles/:article_id", () => {
       });
   });
   test("Status:404 returns an error message if article is not found", () => {
-    //  const requestBody = { inc_votes: 0 };
-    return (
-      request(app)
-        .patch("/api/articles/888")
-        // .send(requestBody)
-        .expect(404)
-        .then(({ body }) => {
-          expect(body.msg).toBe("Article ID 888 does not exist");
-        })
-    );
+    const requestBody = { inc_votes: 0 };
```

Comment:
Please avoid working on other tickets at the same time. This should have been handled in the patch ticket